core/qt-ble.cpp

```diff
@@ -371,6 +364,14 @@ dc_status_t BLEObject::poll(int timeout)
 	return DC_STATUS_SUCCESS;
 }
 
+dc_status_t BLEObject::purge(dc_direction_t direction)
+{
+	if (direction & DC_DIRECTION_INPUT) {
```

Comment:
The [coding style](https://github.com/subsurface/subsurface/blob/master/CODINGSTYLE.md) in Subsurface is that `if` with a single expression does not use curly braces.